osltweaks.php

```diff
@@ -49,22 +49,17 @@ function osltweaks_civicrm_buildForm($formName, &$form) {
     if (array_key_exists($form->_id, ($settings['member_redirect_contribution_pages'] ?? array()))) {
       // Get current contact id
       $currentContactId = CRM_Core_Session::singleton()->getLoggedInContactID();
-        // Get memberships base on the current contact id
-      $memberships = civicrm_api3('Membership', 'get', [
-        'sequential' => 1,
+      // Get memberships count base on the current contact id
+      $memberships = civicrm_api3('Membership', 'getcount', [
```

Comment:
@lucky091588  Since this is an integer count of memberships (not actual membership records), please name this variable $membershipCount.